Version 15.0.1

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am getting the following error with pacote@15.0.1.

npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /Users/xxxx/.npm/_npx/67793/lib/node_modules/pacote/node_modules/@npmcli/installed-package-contents/index.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/xxxx/.npm/_npx/67793/lib/node_modules/pacote/node_modules/@npmcli/installed-package-contents/index.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/xxxx/.npm/_logs/2022-10-18T04_25_34_732Z-debug.log
Install for [ 'pacote@latest' ] failed with code 254


### Expected Behavior

pacote@15.0.0 works fine.


### Steps To Reproduce

1. Run "npx pacote"

### Environment

- npm: 6.14.16
- Node: 14.19.0
- OS: macOS 12.4
- platform: Macbook Pro

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
   ],
   "dependencies": {
     "@npmcli/git": "^4.0.0",
-    "@npmcli/installed-package-contents": "^2.0.0",
+    "@npmcli/installed-package-contents": "^2.0.1",
     "@npmcli/promise-spawn": "^4.0.0",
     "@npmcli/run-script": "^5.0.0",
     "cacache": "^17.0.0",

